backend/src/main/java/com/festago/entryalert/application/EntryAlertEventListener.java

```diff
@@ -0,0 +1,57 @@
+package com.festago.entryalert.application;
+
+import com.festago.entryalert.dto.EntryAlertResponse;
+import com.festago.ticket.dto.event.TicketCreateEvent;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.TaskScheduler;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.event.TransactionPhase;
+import org.springframework.transaction.event.TransactionalEventListener;
+
+@Component
+public class EntryAlertEventListener {
+
+    private static final Logger log = LoggerFactory.getLogger(EntryAlertEventListener.class);
+
+    private final EntryAlertService entryAlertService;
+    private final TaskScheduler taskScheduler;
+    private final Clock clock;
+
+    public EntryAlertEventListener(EntryAlertService entryAlertService, TaskScheduler taskScheduler, Clock clock) {
+        this.entryAlertService = entryAlertService;
+        this.taskScheduler = taskScheduler;
+        this.clock = clock;
+    }
+
+    @EventListener(ApplicationReadyEvent.class)
+    public void initEntryAlertSchedule() {
+        List<EntryAlertResponse> entryAlerts = entryAlertService.findAllPending();
+        entryAlerts.forEach(this::addSchedule);
+    }
+
+    private void addSchedule(EntryAlertResponse entryAlert) {
+        Long entryAlertId = entryAlert.id();
+        log.info("add entryAlert schedule: {}", entryAlertId);
+        Instant alertTime = toInstant(entryAlert.alertTime());
+        taskScheduler.schedule(() -> entryAlertService.sendEntryAlert(entryAlertId), alertTime);
+    }
```

Comment:
```suggestion
    private void addSchedule(EntryAlertResponse entryAlert) {
        Long entryAlertId = entryAlert.id();
        log.info("EntryAlert 스케쥴링 추가. entryAlertId: {}, alertTime: {}", entryAlertId, entryAlert.alertTime());
        Instant alertTime = toInstant(entryAlert.alertTime());
        taskScheduler.schedule(() -> entryAlertService.sendEntryAlert(entryAlertId), alertTime);
    }
```
다음과 같이 자세히 로그를 적어도 좋겠네요.
`alertTime` 변수를 사용하지 않은 이유는 `Instant` 타입이라, UTC 시간(우리나라 시간 -9시간)으로 찍혀 헷갈릴 수 있겠더라구요.